### DISCUSSION TITLE
期待「Tsrpc-Admin 」的诞生

### DISCUSSION THREAD

**Discussion by Dssssds** [k8w/tsrpc#22] [Ideas]
Tsrpc-admin 开发绝对爽歪歪
- [ ] ANTD
- [ ] Seme Design
- [ ] Next.JS

**Comment by k8w** [comment]
啊哈哈，有需要可以先参考：
https://github.com/k8w/tsrpc-antd-pro

ANTD Pro + TSRPC 的例子

主要还是时间原因，目前还有更高优先级的 TODO List 待办~
哈哈也欢迎社区小伙伴共创行业解决方案，有需要随时提供支持~
